<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>KST_EXPORT</exportmacro>
 <class>KstEditViewObjectDialog</class>
 <widget class="QDialog" name="KstEditViewObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>426</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="1" rowspan="4">
    <widget class="EditMultipleWidget" name="_editMultipleWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>241</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_editMultiple">
       <property name="text">
        <string>Edit &amp;Multiple &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>142</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_OK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_apply">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Q3ButtonGroup" name="_propertiesGroup">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="Q3Frame" name="_propertiesFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="spacer5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>41</width>
       <height>210</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="Q3ButtonGroup" name="_buttonGroupDefaults">
     <property name="title">
      <string>Object defaults:</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="_pushButtonSetDefaults">
        <property name="text">
         <string>Set</string>
        </property>
        <attribute name="buttonGroup">
         <string/>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_pushButtonRestoreDefaults">
        <property name="text">
         <string>Restore</string>
        </property>
        <attribute name="buttonGroup">
         <string/>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="spacer4_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>181</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditMultipleWidget</class>
   <extends>QWidget</extends>
   <header>editmultiplewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_propertiesFrame</tabstop>
  <tabstop>_OK</tabstop>
  <tabstop>_apply</tabstop>
  <tabstop>_cancel</tabstop>
 </tabstops>
 <includes>
  <include location="global">kst_export.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
